### ISSUE TITLE
[Enhancement] Install script should open ports 443 and 3128 before bringing up registry server

### ISSUE THREAD

**Issue by sameerjvn-old** [oneconvergence/registry-server#3]
- Registry server runs on port 443
- Proxy server runs on port 3128
- install.sh script should automatically open those ports as part of the script.

**Comment by hemanthravi** [comment]
@deepio-oc 
- I don't think we should open ports automatically.
- Document these ports as part of prerequisites for user to open before running install

**Comment by deepio-oc** [comment]
yes, you are right.  it may be a security liability if we open ports behind the scene.

@sameerjvn once these ports are documented, you can close this issue.
